### PR TITLE
Add heaptoptions for connect, ksql and schema-registry

### PR DESCRIPTION
**Pull request by nysthee**
As per: https://github.com/confluentinc/cp-helm-charts/issues/130

Would like some feedback for the defaults.

### CODE REVIEW

File: charts/cp-ksql-server/templates/deployment.yaml [hunk 1]
```diff
@@ -76,6 +76,8 @@ spec:
             value: {{ template "cp-ksql-server.serviceId" . }}
           - name: KSQL_KSQL_SCHEMA_REGISTRY_URL
             value: {{ template "cp-ksql-server.cp-schema-registry.service-name" . }}
+           - name: KAFKA_HEAP_OPTS
```
Comment (qshao-pivotal) [inline charts/cp-ksql-server/templates/deployment.yaml]: Please remove the extra space. It will cause formatting error.

**Comment by qshao-pivotal** [comment]
Actually I don't think KAFKA_HEAP_OPT works on these components.

**Comment by nysthee** [comment]
For schema-registry: https://github.com/confluentinc/schema-registry/blob/8f727d74da65a9be4f7fd82fd8a2498ad48c85ec/bin/windows/schema-registry-run-class.bat#L59

For KSQL: https://github.com/confluentinc/ksql/blob/54dedea567dc5994ad46fcbde41903f83dfe7782/bin/ksql-run-class#L83

For connect: https://github.com/confluentinc/cp-docker-images/issues/593

**Comment by nysthee** [comment]
I was a bit to assuming they were all the same :)

**Comment by nysthee** [comment]
I think I have it all covered :)

**Comment by qshao-pivotal** [comment]
LGTM. Thanks!

**Pull request merged**